website_product_filters/models/product.py

```diff
@@ -80,11 +80,10 @@ class ProductCategory(models.Model):
         "product_template_id", readonly=True)
     total_tree_products = fields.Integer("Total Subcategory Prods",
                                          compute="_get_product_count",
-                                         store=True,)
+                                         store=False,)
```

Comment:
@josemoralesp I made this change, when I coded this it was intended to be ```store=False``` do you have any problem with this?